Purchases/Purchasing/RCAttributionFetcher.m

```diff
@@ -51,7 +81,8 @@ - (nullable NSString *)identifierForAdvertisers {
         NSString *mangledIdentifierPropertyName = @"nqiregvfvatVqragvsvre";
 
         NSString *className = [self rot13:mangledClassName];
-        id <FakeASIdentifierManager> asIdentifierManagerClass = (id <FakeASIdentifierManager>) NSClassFromString(className);
+        id <FakeASIdentifierManager>
+            asIdentifierManagerClass = (id <FakeASIdentifierManager>) NSClassFromString(className);
```

Comment:
Is that line break normal? Shouldn't this be:

```
        id <FakeASIdentifierManager> asIdentifierManagerClass = 
                (id <FakeASIdentifierManager>) NSClassFromString(className);
```

Reply:
good catch